examples/audio/sft/asr/aishell/path.sh

```
@@ -1,14 +1,19 @@
-# cuda related
-unset LD_LIBRARY_PATH
-cuda_prefix=/bucket/output/jfs-hdfs/user/xingchen.song/tools/cuda
-cache_prefix=/bucket/output/jfs-hdfs/user/xingchen.song/share
-cuda_version=12.6.3
-driver_version=560.35.05
-cudnn_version=9.5.1.17
+cuda_prefix=/usr/local
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)